contribute/contribute.rst

```diff
@@ -344,10 +344,13 @@ In our example setup, the URL from **laptop** would be :code:`https://buildvm/`.
 
 It is very important to ensure that your code changes survive a reboot.
 Sometimes, especially when database changes are made, this can be an issue.
-Be sure to check that the resulting build behaves as expected over:
+Be sure to also check that the resulting build behaves as expected over a hard-reset:
 
-1. A config reset - removing :code:`/opt/rockstor/.initrock` and rebooting
-2. Several reboots
+  1. Do a total config reset by removing :code:`/opt/rockstor/.initrock` and rebooting.
+  2. Redo the :ref:`webui_setup` :code:`userdel username` first if reusing username.
```

Comment:
How about recommending to always delete the user anyway?
```suggestion
  2. Delete the Rockstor admin user: :code:`userdel username`.
  3. Redo the :ref:`webui_setup` .
```

Reply:
@FroggyFlox Super fast review here.
Re the suggested change of breaking out the delete of the OS side of the Web-UI admin user: I had thought of this but settled on the crunched up 2 steps. Lets go with your suggestion. It can only error out harmlessly if they don't exist. I was considering folks creating additional users but that is not the way to go anyway so yes.

Reply:
Oh I didn't think of the username not existing... good point. It's true it doesn't hurt, though; I personally often fall in that case when O forget whether I already deleted it :-p